### DISCUSSION TITLE
How to modeling a missile launch in UE5

### DISCUSSION THREAD

**Discussion by eusru** [JSBSim-Team/jsbsim#701] [Q&A]
I am new to JSBSim. 
My question is how to launch a missile in my scene by given a start position and  a end position.
My target effect is that the missile uses a real aerodynamic model, and after inputting some parameters (mainly the mass, speed, initial point position, target point position, etc.), the missile truly simulates the flight trajectory in ue5

**Comment by bcoconni** [comment]
As far as the flight model is involved, you'll need to model a vehicle such as the [J246 rocket](https://github.com/JSBSim-Team/jsbsim/blob/master/aircraft/J246/J246.xml) as well as the autopilot that will fly the missile to the target.
